src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -9,7 +9,10 @@ import MPulseBeacon from '../containers/MPulseBeacon';
 import { ServiceContext } from '../contexts/ServiceContext';
 
 const PageWrapper = ({ children }) => {
-  const { fonts } = useContext(ServiceContext);
+  const { fonts: fontFunctions } = useContext(ServiceContext);
+
+  const fonts = fontFunctions.map(func => func());
```

Comment:
I was about to mention that the font function needs a baseUrl argument but then saw in the psammead package that it defaults to one

```js
export const F_REITH_SERIF_REGULAR = baseUrlOverride => `
  @font-face {
    font-family: "ReithSerif";
    src: url("${baseUrlOverride ||
      baseFontUrl}BBCReithSerif_W_Rg.woff2") format("woff2"), url("${baseUrlOverride ||
  baseFontUrl}BBCReithSerif_W_Rg.woff") format("woff");
    font-display: optional;
  }`;
```

it would be much clearer if the default was assigned to the argument:

```js
export const F_REITH_SERIF_REGULAR = (baseUrlOverride = baseFontUrl) => `
  @font-face {
    font-family: "ReithSerif";
    src: url("${baseUrlOverride}BBCReithSerif_W_Rg.woff2") format("woff2"),
         url("${baseUrlOverride}BBCReithSerif_W_Rg.woff") format("woff");
    font-display: optional;
  }`;
```

Reply:
Resolving in this pr https://github.com/bbc/psammead/pull/2307